Fix issues in NFT tutorial - Closes #1480 (#1482)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/tutorial/nft.adoc

* `registeredNFTTokensSchema`: The schema, how the NFTs are saved in the database.
Each NFT has the following properties:
** id: The unique ID of the NFT.
** value: The value of the NFT.
** ownerAddress: The address of the owner of the NFT.
** minPurchaseMargin: The minimum price increase, that should happen with every purchase of the NFT in percentage.
** name: The NFT name.
* `CHAIN_STATE_NFT_TOKENS`: The key under which the NFTs are saved in the database.

> * `createNFTToken()`: Creates a new NFT based on the provided `name`, `ownerAddress`, `nonce`, `value` and  `minPurchaseMargin`.

Reviewer: ```suggestion
* `createNFTToken()`: Creates a new NFT based on the provided `name`, `ownerAddress`, `nonce`, `value`, and  `minPurchaseMargin`.
```

Analog to the implementation of the `createNFT` asset, create the `purchaseNFT` asset by pasting the snippet below.

> NOTE: The validation of the asset inputs is done in the `apply()` function, because we need to access the database to validate the transaction inputs.

Reviewer: ```suggestion
NOTE: The validation of the asset inputs is done in the `apply()` function, as it is necessary to access the database in order to validate the transaction inputs.
```
